extensions/sql-database-projects/src/test/baselines/openNewStyleSqlProjectWithFilesSpecifiedBaseline.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build">
+  <Sdk Name="Microsoft.Build.Sql" Version="1.0.0" />
+  <PropertyGroup>
+    <Name>TestProjectName</Name>
+    <ProjectGuid>{2C283C5D-9E4A-4313-8FF9-4E0CEE20B063}</ProjectGuid>
+    <DSP>Microsoft.Data.Tools.Schema.Sql.Sql150DatabaseSchemaProvider</DSP>
+    <ModelCollation>1033, CI</ModelCollation>
+  </PropertyGroup>
+  <Target Name="BeforeBuild">
+    <Delete Files="$(BaseIntermediateOutputPath)\project.assets.json" />
+  </Target>
+  <ItemGroup>
+    <Folder Include="Properties" />
+    <Folder Include="folder1" />
+  </ItemGroup>
+  <ItemGroup>
+    <Build Include="folder1\file2.sql" />
```

Comment:
The Include/Remove also support wild cards too like `<Build Include="folder1\*.sql" />` can you verify this works too?

Reply:
added a test for a few of those patterns